README.md

```diff
@@ -160,6 +160,25 @@ QubitSDK.tracker().getExperiences(
 
 where `variation`, `preview`, `ignoreSegments` are optional parameters
 
+# Placements
+
+SDK contains method to fetch the Placements.
+
+Kotlin snippet:
+
+```kotlin
+QubitSDK.tracker().getPlacement(
```

Comment:
Is getting placement element of event tracker? 
I know, that experiences was done the same way. I had same doubts, when I discover it long time after finishing implementation.

Reply:
You're right, this should be placed in QubitSDK class. Moving there.

I'm keeping Experience* classes as they are in order not to break backward compatibility.